test/integration/ComputeFlow.test.ts

```diff
@@ -826,6 +930,13 @@ describe('Compute flow', () => {
         const balance = await datatoken.balance(tokenAddressAdditional2, bob.getId())
         assert(balance.toString() === dTamount.toString())
       })
+
+    await datatoken
+      .transfer(tokenAddressWithCustomData, bob.getId(), dTamount, alice.getId())
+      .then(async () => {
```

Comment:
not fully sure this has any real implication, but the mix of async/await with promise.then seems unnecessary. This is the same following the established async/await syntax in this file:

```js
await datatoken.transfer(tokenAddressWithCustomData, bob.getId(), dTamount, alice.getId())
const balance = await datatoken.balance(tokenAddressWithCustomData, bob.getId())
assert(balance.toString() === dTamount.toString())
```